kinds/role/role_kind.cue

```diff
@@ -0,0 +1,28 @@
+package kind
+
+name:        "Role"
+maturity:    "merged"
+description: "Roles represent a set of users+teams that should share similar access"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				spec: {
+					// The role identifier `managed:builtins:editor:permissions`
+					name: string
+					// Optional display
+					displayName?: string
+					// Name of the team.
```

Comment:
This appears to be used to group roles together in the ui

Reply:
Also looks like it is derived from upstream systems